app/src/js/middleware/request.js

```diff
@@ -10,30 +7,28 @@ import {
 import log from '../utils/log';
 import { isValidApiRequestAction } from './validate';
 
-const handleError = ({ id, type, error, requestAction }, next) => {
+// Use require to allow for mocking
+const requestPromise = require('request-promise');
+const { loginError } = require('../actions');
+
+const handleError = ({
+  id,
+  type,
+  error,
+  requestAction,
+  statusCode
+}, next) => {
   console.groupCollapsed('handleError');
   console.log(`id: ${id}`);
   console.log(`type: ${type}`);
   console.dir(error);
   console.dir(requestAction);
   console.groupEnd();
-  if (error.message) {
```

Comment:
All of this code is major technical debt and as far as I can tell was untested anywhere. So I ripped it out in favor of the change to log out the user when we receive specific error response codes from the API